### DISCUSSION TITLE
Refund for recent purchase.

### DISCUSSION THREAD

**Discussion by Devcharij** [aclap-dev/video-downloadhelper#1324] [License & Purchase requests]
I am writing to formally request a refund for the recent purchase made under my account. Below are the details of the transaction:

- **email:** jcaris.94[at]gmail.com

I kindly ask you to process this refund at your earliest convenience. Please let me know if you require any further information to complete this request.

Thank you for your attention to this request. I look forward to your prompt response.

Best regards,

**Comment by jcfrog** [comment]
Hi,
You have been refunded. Should be available in a few business days.
